### ISSUE TITLE
Run the interop tests on jdk11

### ISSUE THREAD

**Issue by raboof** [akka/akka-grpc#530]
Since ALPN is now included in the JDK, setting up the interop tests would have to be different when running on that.

**Comment by dwijnand** [comment]
Some progress in #551.

**Comment by dwijnand** [comment]
Failures on Java 11:

```
[0m[[0m[31merror[0m] [0m[0mFailed tests:[0m
[0m[[0m[31merror[0m] [0m[0m	akka.grpc.interop.GrpcInteropAkkaScalaWithIoSpec[0m
[0m[[0m[31merror[0m] [0m[0m	akka.grpc.interop.GrpcInteropAkkaJavaWithIoSpec[0m
[0m[[0m[31merror[0m] [0m[0m	akka.grpc.interop.GrpcInteropAkkaJavaWithAkkaScalaSpec[0m
[0m[[0m[31merror[0m] [0m[0m	akka.grpc.interop.GrpcInteropAkkaJavaWithAkkaJavaSpec[0m
[0m[[0m[31merror[0m] [0m[0m	akka.grpc.interop.GrpcInteropAkkaScalaWithAkkaScalaSpec[0m
[0m[[0m[31merror[0m] [0m[0m	akka.grpc.interop.GrpcInteropAkkaScalaWithAkkaJavaSpec[0m
[0m[[0m[31merror[0m] [0m[0m(akka-grpc-interop-tests / Test / [31mtest[0m) sbt.TestsFailedException: Tests unsuccessful[0m
[0m[[0m[31merror[0m] [0m[0mTotal time: 238 s, completed Mar 25, 2019, 10:32:52 AM[0m
```

(from https://api.travis-ci.org/v3/job/510854812/log.txt #551's merge commit)

**Comment by raboof** [comment]
Fixed in #551 and #568
